packages/material/src/util/datejs.ts

```diff
@@ -8,13 +8,13 @@ export const createOnChangeHandler = (
   path: string,
   handleChange: (path: string, value: any) => void,
   saveFormat: string | undefined
-) => (time: dayjs.Dayjs) => {
+) => (time: dayjs.Dayjs, keyboardInputValue: string) => {
```

Comment:
looks good, I would just rename the parameter
```suggestion
) => (time: dayjs.Dayjs, textInputValue: string) => {
```

Reply:
This name comes from MUI, but they're not always great at names and `textInputValue` makes more sense. Will update this.